CrisisTextLine/Elmer
# Get volunteer matches

#### What's this PR do?
Retrieves information for all volunteers that are interested in VM and adds them to a MailChimp List
#### Where should the reviewer start?
#### How should this be manually tested?
1) Add global.Keys and .config
2) run the file
3) check that the .csv was created correctly
4) users listed in the csv should all be present in MailChimp List: https://us5.admin.mailchimp.com/lists/members/?id=189493#p:1-s:100-so:false

#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-323
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-409
#### Questions:
